src/AutoMerge/github.jl

```diff
@@ -176,6 +178,21 @@ function merge!(registry_repo::GitHub.Repo,
     return nothing
 end
 
+function wait_pr_compute_mergeability(repo::GitHub.Repo,
+                                      pr::GitHub.PullRequest;
+                                      auth::GitHub.Authorization)
+    sleep(5)
+    max_tries = 10
+    num_tries = 0
+    pr = GitHub.pull_request(repo, pr.number; auth = auth)
```

Comment:
This call might not be needed if the PR is already mergeable?

Reply:
So, here's the problem.

Suppose we have two PR's, PR1 and PR2.

We have a vector containing both of those PR's, and we iterate through the vector.

First we process PR1, and decide to merge it, so we merge it.

Merging PR1 might change whether or not PR2 is mergeable. But when we merge PR1, it doesn't update our local copy of the PR2 object. So we have to re-fetch PR2 from the GitHub API to see if PR2 is still mergeable after PR1 is merged.

Reply:
Yea, just realized.